src/components/feedback/Modal.tsx

```diff
@@ -138,11 +143,12 @@ const ModalNext = function Modal({
       >
         <Panel
           buttons={buttons}
+          fullWidthHeader={true}
```

Comment:
Previously, all Modals were `scrollable`, and `Panel` will set a full-width header on when `scrollable`. To avoid confusion or inconsistency, all Modals will continue to have a full-width header whether or not they are `scrollable`.